source/application/AMPIC887DriverServer/AMPIC887Controller.h

```diff
@@ -0,0 +1,82 @@
+#ifndef AMPIC887CONTROLLER_H
+#define AMPIC887CONTROLLER_H
+
+#define CONTROLLER_PORT 50000 // PI C887.11 controller are hardwired to use port 50000
+
+#include <QString>
+#include "GCS2Commands/AMGCS2Command.h"
+
+
+/*!
+  * A class representing a single PI C887.11 Controller. Commands and queries issued
+  * to instances of this class will, in turn, pass them along to the controller
+  * they represent.
+  */
+class AMPIC887Controller
+{
+public:
+	/*!
+	  * Creates a new instance of an C887.11 controller, with the provided hostname
+	  * and id. The controller will not be connected until the connect() call has
```

Comment:
How relevant is the comment _The controller will not be connected until the connect() call has been made._ when you call connect inside the constructor?